test/index.js

```diff
@@ -1243,6 +1251,29 @@ foo :: { x :: a, y :: a } -> { x :: a, y :: a }
 2)  123 :: Number
 
 Since there is no type of which all the above values are members, the type-variable constraint has been violated.
+`));
+
+    //    fooBarBaz :: { 'foo "bar" baz' :: Number } -> Number
+    const fooBarBaz =
+    def ('fooBarBaz')
+        ({})
+        ([$.RecordType ({'foo "bar" baz': $.Number}), $.Number])
+        (r => r['foo "bar" baz']);
+
+    eq (fooBarBaz ({'foo "bar" baz': 42})) (42);
+
+    throws (() => fooBarBaz ({'foo "bar" baz': null}))
+           (new TypeError (`Invalid value
+
+fooBarBaz :: { "foo \\"bar\\" baz" :: Number } -> Number
+                                    ^^^^^^
+                                      1
```

Comment:
The alignment here looks wrong, but that's because `\\` renders as a single backslash.